c_binding/include/libstoragemgmt/libstoragemgmt_local_disk.h

```diff
@@ -104,6 +104,56 @@ int LSM_DLL_EXPORT lsm_local_disk_vpd83_get(const char *sd_path,
 int LSM_DLL_EXPORT lsm_local_disk_rpm_get(const char *disk_path, int32_t *rpm,
                                           lsm_error **lsm_err);
 
+/**
+ * Query the disk link type of given disk path.
+ * For SATA disks connected to SAS SES enclosure, will return
+ * LSM_SCSI_LINK_TYPE_ATA.
+ * Require permission to open /dev/sdX(root user or disk group).
+ * New in version 1.3.
+ * @param[in]  disk_path    String. The path of disk, example "/dev/sdb".
+ * @param[out] link_type    Output pointer of lsm_disk_link_type.
+ *                          LSM_DISK_LINK_TYPE_UNKNOWN when error.
+ *                          Possible values are:
+ *                              LSM_DISK_LINK_TYPE_UNKNOWN
+ *                                  Unknown
+ *                              LSM_DISK_LINK_TYPE_FC
+ *                                  Fibre Channel
+ *                              LSM_DISK_LINK_TYPE_SSA
+ *                                  Serial Storage Architecture, Old IBM tech.
+ *                              LSM_DISK_LINK_TYPE_SBP = 3,
```

Comment:
The first enumerated types have no associated numerical value, but the rest do.  Please make them all the same, one way or the other.